pkg/api/router.go

```diff
@@ -208,6 +208,16 @@ func (s *Service) checkChequebookAvailability(handler http.Handler) http.Handler
 	})
 }
 
+func (s *Service) checkStorageIncentivesAvailability(handler http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if s.redistributionAgent == nil {
+			jsonhttp.NotImplemented(w, "Storage incentives are disabled. This endpoint is unavailable.")
```

Comment:
well, it is implemented, it is more like a `404 Not Found` or a `403 Forbidden`. I would simply go with 404

Reply:
403 is also fine from my side on all similar places

Reply:
tests still need to be adjusted: https://github.com/ethersphere/bee/actions/runs/16437874357/job/46453779979?pr=5150